apps/src/templates/certificates/Certificate.jsx

```diff
@@ -131,53 +138,73 @@ function Certificate(props) {
   const print = getPrintPath();
 
   return (
-    <div style={styles.container}>
-      <h1 style={headingStyle}>{i18n.congratsCertificateHeading()}</h1>
+    <div className={style.container}>
+      <div className={style.headerContainer}>
+        <Heading1 className={`${headingStyle} ${style.header}`}>
+          {i18n.congratsCertificateHeading()}
+        </Heading1>
+      </div>
       {tutorial && (
         <LargeChevronLink
           link={`/s/${tutorial}`}
           linkText={i18n.backToActivity()}
         />
       )}
-      <div id="uitest-certificate" style={certificateStyle}>
-        <BackToFrontConfetti active={personalized} style={styles.confetti} />
-        <a href={certificateShareLink}>
-          <img src={imgSrc} />
-        </a>
-      </div>
-      <div style={certificateStyle}>
-        {tutorial && !personalized && (
-          <div>
-            <h2>{i18n.congratsCertificatePersonalize()}</h2>
-            <input
-              id="name"
-              type="text"
-              style={styles.nameInput}
-              placeholder={i18n.yourName()}
-              ref={nameInputRef}
-            />
-            <button
-              type="button"
-              style={styles.submit}
-              onClick={personalizeCertificate.bind(this, certificateId)}
-            >
-              {i18n.submit()}
-            </button>
-          </div>
-        )}
-        {tutorial && personalized && (
-          <div>
-            <h2 id="uitest-thanks">{i18n.congratsCertificateThanks()}</h2>
-            <p>{i18n.congratsCertificateContinue()}</p>
-          </div>
-        )}
-        <h2>{i18n.congratsCertificateShare()}</h2>
-        <SocialShare
-          facebook={facebook}
-          twitter={twitter}
-          print={print}
-          under13={under13}
-        />
+      <div className={style.certificateContainer}>
+        <div id="uitest-certificate" className={certificateStyle}>
```

Comment:
Same for this ui test ID

Reply:
Same goes here. Id is Being used within `hour_of_code_finish.feature`.